tests/serverpod_test_server/test_integration/websockets/websocket_extensions.dart

```diff
@@ -0,0 +1,10 @@
+import 'dart:async';
+import 'package:web_socket/web_socket.dart';
+
+/// Extension to provide backward-compatible stream-based API for WebSocket.
+extension WebSocketExtensions on WebSocket {
+  Stream<String> get textEvents => events
+      .where((event) => event is TextDataReceived)
+      .cast<TextDataReceived>()
+      .map((e) => e.text);
```

Comment:
I thought this should be:
```suggestion
      .whereType<TextDataReceived>()
      .map((e) => e.text);
```

Reply:
Are yes, I [forgot](https://github.com/dart-lang/sdk/issues/34050)